apps/frontend/src/components/listingCard.tsx

```diff
@@ -0,0 +1,110 @@
+import { Grid, Button, Typography } from "@mui/material";
+import PropTypes from "prop-types";
+
+interface ListingObject {
+  listingID: string;
+  sellerID: string;
+  sellerName: string;
+  title: string;
+  description: string;
+  price: number;
+  dateCreated: string;
+  imageUrl: string;
+}
+
+interface ListingCardProps {
+  listing: ListingObject;
+}
+
+const priceFormatter = new Intl.NumberFormat("en-CA", {
+  style: "currency",
+  currency: "CAD",
+});
+
+const convertDate = (dateString: string) => {
+  const dateTimeObject = new Date(dateString);
+  return dateTimeObject.toDateString();
+};
+
+const ListingCard: React.FC<ListingCardProps> = ({ listing }) => {
+  return (
+    <Grid
```

Comment:
I don't think Grid should be used here unless it's actually applying grid styling. I'm not super familiar with MUI, but I'd assume Box is the better choice

Reply:
There's a few places where this occurs, but I'm just gonna leave this one comment for all of them

Reply:
For this one specifically the grid worked best to make sure that all the information stayed in the same spot but there are some others that Ill change to boxes since I dont need the grid styling